image_client/csv_purger.py

```diff
@@ -0,0 +1,113 @@
+import picturae_config as picturae_config
+from importer import Importer
+import traceback
+import pandas as pd
+
```

Comment:
Comment here -what does this class do?

Reply:
.. also, I'm not sure "csvdatabase" is a good class name. I'm guessing this reads CSV files from picturae and creates specify database records? so maybe "read_picturae_csv" or something like that..?